config/structs.go

```diff
@@ -4,6 +4,7 @@ type ScheduledFeedConfig struct {
 	PubConfig    PublisherConfig `yaml:"publisher"`
 	EnabledFeeds []string        `yaml:"enabled_feeds"`
 	HttpPort     int             `yaml:"http_port,omitempty"`
+	CutoffDelta  string          `yaml:"time_delta"`
```

Comment:
time_delta and cutoff_delta are used in different parts of this PR, these will need to be updated to be consistent.

Reply:
Ah sorry, missed replacing that one